### PR TITLE
Fix or remove already fixed TODOs in code

### PR DESCRIPTION
**Pull request by axbarsan**
Some of the older TODOs can be easily fixed, or even removed, since some of them have been already fixed.

### CODE REVIEW

File: src/actions/clusterActions.js [hunk 1]
```diff
@@ -305,9 +305,6 @@ function clusterLoadStatus(clusterId, { withLoadingFlags }) {
         return status; // used in clusterLoadDetails!
       })
       .catch((error) => {
-        // TODO: Find a better way to deal with status endpoint errors in dev:
```
Comment (axbarsan) [inline src/actions/clusterActions.js:308]: Issue has been closed, we no longer care about this

File: src/actions/userActions.js [hunk 2]
```diff
@@ -213,11 +213,6 @@ export function unauthorized() {
     // requests.
     clearQueues();
 
-    // Let the user know he has been logged out due to a probably expired
-    // or deleted token.
-    // TODO: Avoid this message firing multiple times from different queries,
-    // maybe by using a custom queue with length 1.
-    // See https://ned.im/noty/#/api?id=api-static-methods
```
Comment (axbarsan) [inline src/actions/userActions.js:220]: This has been already fixed by https://github.com/giantswarm/happa/pull/1582

File: src/components/Auth/Login.tsx [hunk 3]
```diff
@@ -129,7 +129,6 @@ class Login extends React.Component<ILoginProps, ILoginState> {
     }
   };
 
-  //TODO: turn progressbutton into a component
```
Comment (axbarsan) [inline src/components/Auth/Login.tsx:132]: Already using our `Button` component

File: src/components/Cluster/NewCluster/CreateNodePoolsCluster.js [hunk 4]
```diff
@@ -150,7 +150,6 @@ class CreateNodePoolsCluster extends Component {
     };
 
     if (this.state.hasAZLabels) {
-      // TODO: don't use array here as long as there can be only one master node.
```
Comment (axbarsan) [inline src/components/Cluster/NewCluster/CreateNodePoolsCluster.js:153]: Doesn't really make sense

**Review by oponder — APPROVED** [review]
LGTM if the the cluster detail screen and various cpu / memory displays are still working correctly.

**Pull request merged**